src/librustc/middle/reachable.rs

```diff
@@ -33,6 +33,9 @@ fn item_might_be_inlined(tcx: TyCtxt<'tcx>, item: &hir::Item, attrs: CodegenFnAt
     }
 
     match item.node {
+        hir::ItemKind::Fn(_, header, ..) if header.constness == hir::Constness::Const => {
```

Comment:
```suggestion
        hir::ItemKind::Fn(_, header, ..) if header.is_const() => {
```

(There is some follow-up cleanup potential from using more of `.is_const()` from a quick search `hir::Constness::Const`.)

Reply:
Oh, thanks! Uses it instead of `==`.